docs/execute.adoc

```diff
@@ -111,14 +111,14 @@ Since Saturn is designed around `DLEN`-wide element groups as the base throughpu
 The scheduling mechanism precisely tracks which element groups an instruction or operation has yet to read-or-write to interlock the sequencers.
 
 In Saturn, the "out-of-order instruction window" includes all instructions in the issue queues (but not the VDQ), the instructions currently in progress within the sequencers, and any operations which have not yet completed execution in the functional unit datapaths.
-All instructions in this window must advertise a precise set of element groups they have not yet read or written, along with the age tag of the instruction.
+All instructions in this window must advertise a precise set of element groups that have not yet read or written, along with the age tag of the instruction.
```

Comment:
Can you make this "groups that will be read or written by this instruction in future cycles"